server/app/utils/emailer.py

```diff
@@ -57,11 +50,52 @@ def email_jump_info(jumps: list=None, check_id: int=None)->None:
+            if x[4] > max_jump and x[5] > max_jump:
+                html += "<tr><td class='alnright'>" + str(x[0]) + "</td><td class='alnright'>" + str(x[1])+ "</td><td class='alnright'>" + str(x[2])+"%" + "</td><td class='alnright'>" + str(x[3])+ "%"  + "</td><td class='alnright' style='color:#ea0b22'>" + str(x[4]) +"%"+ "</td><td class='alnright' style='color:#ea0b22'>" + str(x[5]) +"%" + "</td></tr>"
+            elif x[4] > max_jump:
+                html += "<tr><td class='alnright'>" + str(x[0]) + "</td><td class='alnright'>" + str(x[1])+ "</td><td class='alnright'>" + str(x[2])+"%" + "</td><td class='alnright'>" + str(x[3])+ "%"  + "</td><td class='alnright' style='color:#ea0b22'>" + str(x[4]) +"%"+ "</td><td class='alnright'>" + str(x[5]) +"%" + "</td></tr>"
+            elif x[5] > max_jump:
+                html += "<tr><td class='alnright'>" + str(x[0]) + "</td><td class='alnright'>" + str(x[1])+ "</td><td class='alnright'>" + str(x[2])+"%" + "</td><td class='alnright'>" + str(x[3])+ "%"  + "</td><td class='alnright'>" + str(x[4]) +"%"+ "</td><td class='alnright' style='color:#ea0b22'>" + str(x[5]) +"%" + "</td></tr>"
+            else:
+                html += "<tr><td class='alnright'>" + str(x[0]) + "</td><td class='alnright'>" + str(x[1])+ "</td><td class='alnright'>" + str(x[2])+"%" + "</td><td class='alnright'>" + str(x[3])+ "%"  + "</td><td class='alnright'>" + str(x[4]) +"%"+ "</td><td class='alnright'>" + str(x[5]) +"%" + "</td></tr>"
 
         html += """\
                                 </table>
                             </div>
+                            <div>
+                                <p><b>NOTE:</b></p>
+                                <table>
+                                    <tr>
+                                    <th>Column Name</th>
+                                    <th>Description</th>
+                                    </tr>
+                                    <tr>
+                                    <td>Repo A</td>
+                                    <td>Code of Team A being checked</td>
+                                    </tr>
+                                    <tr>
+                                        <td>Repo B</td>
+                                        <td>Code of Team B being checked</td>
+                                    </tr>
+                                    <tr>
+                                        <td>Shared Code Repo A to B</td>
+                                        <td>Percentage of first team A&apos;s code that is shared with second team B for the current run</td>
+                                    </tr>
+                                    <tr>
+                                        <td>Similarity Jump Repo A to B</td>
+                                        <td>Difference between the similarity percentage of Team A with Team B from current run to previous run</td>
```

Comment:
"Difference between the similarity percentage of Team A with Team B from current run to previous run" is kind of difficult to understand.  I suggest something like,
"Change from the similarity of Team A's code with Team B's code in the previous run (positive number means the code is more similar this time)."